Danki2/Assets/Scripts/Actor/Mortal/Components/HealthBar.cs

```diff
@@ -0,0 +1,40 @@
+﻿using UnityEngine;
+using UnityEngine.UI;
+
+public class HealthBar : MonoBehaviour
+{
+    public Image _healthBar;
+    public Mortal _mortal;
+
+    void Update()
+    {
+        if (_healthBar == null || _healthBar.canvas == null)
+        {
+            return;
```

Comment:
Add a Debug.Log here, to avoid searching for an issue in the future if this case is ever met.